lib/engine/devices/builtins/thingengine.builtin.ts

```diff
@@ -227,7 +227,25 @@ export default class MiscellaneousDevice extends Tp.BaseDevice {
     }
 
     do_alert() {
-        return { message_output : 'Time\'s up!'};
+        return { message_output : 'Alert' };
+    }
+
+    do_timer_expire(env : ExecWrapper) {
+        let now = new Date;
```

Comment:
I think by eslint you need to use `const` in a bunch of places. I'm surprised it hasn't complained yet, and I recommend enabling eslint checking in your IDE (in VSCode, you need to enable the eslint extension and then authorize executing code in the project when the popup is shown)